backstop/tests/skip-to-content.html

```diff
@@ -0,0 +1,53 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>skip-to-content | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body
+    class="iui-body"
+    style="position: relative"
+  >
+    <a
+      href="#demo-skip-to-here"
+      class="iui-skip-to-content"
+      id="test-1"
+    >Skip to main content</a>
+    <theme-button></theme-button>
+    <h1>Skip to content</h1>
+    <hr />
+
+    <section id="other-content">
+      <div style="height: 1000px; border: solid 1px;">
+        <p>This is not the main content.</p>
```

Comment:
Maybe add some interactable elements (buttons, links, etc) so that it's obvious the skip link even works

Reply:
Maybe make it more realistic? 😄 but upto you

Reply:
Done